tools/bridge/tests/test_confirmation_sender.py

```diff
@@ -151,7 +171,9 @@ def test_transaction_sending(
     validator_address,
 ):
     transaction = confirmation_sender.sender.prepare_confirmation_transaction(
-        transfer_event, nonce=confirmation_sender.sender.get_next_nonce()
+        transfer_event,
+        nonce=confirmation_sender.sender.get_next_nonce(),
+        chain_id=int(w3_home.net.version),
```

Comment:
@schmir-at-bb This will most likely not work with the current parity version we use. Please have a chat with Come about it.

Reply:
@cducrest Can you please take a look at this commit?

What I'm doing here is signing a transaction. I get the chainid initially with
```
self.chain_id = int(self.w3.net.version)
```
and provide that as a chainid, when signing a transaction.

This looks like it's related to the PR you created for web3, where the validate_chain_id has been changed to compare the chainid against web3.eth.chainId instead of web3.net.version.

I did have web3.eth.chainId in the beginning, but I saw a deprecation warning about that (probably suggesting to use web3.net.version).

At the moment I think I should have kept using web3.eth.chainId as chainid.

Reply:
Afaik, `web3.net.chainId` should give a deprecation warning since it was actually wrong (it was the network id) and it is replaced with `web3.net.version`. You should use `web3.eth.chainId`, and this should not give you a deprecation warning.

I am too lazy to pull your PR to try.

Reply:
Thanks for the explanation.

Reply:
This will be fixed with commit 15613e5d79bea965944989e4337351a
(which still has to be merged). I didn't go through adding the fix in this PR, because it would result in merge conflicts.

Reply:
good for me